packages/data/src/api/contractMinters/contractMinters.ts

```diff
@@ -0,0 +1,24 @@
+import { Network } from '@mintbase-js/sdk';
+import { fetchGraphQl } from '../../graphql/fetch';
+import { ParsedDataReturn } from '../../types';
+import { parseData } from '../../utils';
+import { contractMintersQuery } from './contractMinters.query';
+import { ContractMintersResults } from './contractMinters.types';
+
+
+export const contractMinters = async (
+  contractAddress: string | string[],
+  network?: Network,
+): Promise<ParsedDataReturn<ContractMintersResults>> => {
```

Comment:
It would be more straight-forward for users if they just got a `string[]` as return.